liquidprompt

```diff
@@ -3398,6 +3465,7 @@ _lp_fossil_unstaged_files() { return 2 ; }
 _lp_fossil_unstaged_lines() { return 2 ; }
 _lp_fossil_staged_files() { return 2 ; }
 _lp_fossil_staged_lines() { return 2 ; }
+_lp_fossil_remote() { return 2 ; }
```

Comment:
These Bazzar functions are all wrong. I messed it up when adding these not implemented functions back in 4adbbf2c. If you could fix them as part of this that would be great.
```suggestion
_lp_bzr_remote() { return 2 ; }
```